### PR TITLE
docs: update README for running e2e tests locally

### PR DESCRIPTION
**Pull request by rabi-siddique**
The PR documents the process of running e2e tests locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,21 +26,67 @@ There are two ways to run the tests:
...
+To run tests on your local machine, first you need to start the development server:
+
+```bash
 yarn build && yarn preview
 ```
 
-Then you need to run the tests using
+### 2. Ensure Keys are in Local Keyring
+
+You need to ensure that the key `user1` is present in your local keyring. To find the mnemonic for this key, open the file `test/e2e/test.utils.js`. On `line 1` of this file, you will find an object named `mnemonics`, which contains the mnemonics for different keys, including `user1` and `user2`.
```
Comment (Muneeb147) [inline README.md:39]: minor:
We can remove "On `line 1` of this file" statement.
Could be a case when someone changes the file `test/e2e/test.utils.js` and `mnemonics` jumps to another line.

File: test/e2e/test.utils.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const config = {
     COMMAND_TIMEOUT: 2 * 60 * 1000,
   },
   testnet: {
-    userKeyName: 'user2',
+    userKeyName: 'user1',
```
Comment (Muneeb147) [inline test/e2e/test.utils.js:63]: Why user1? Addresses and mnemonics are of user2 here 🤔
Does our test config is setup according to user1 only?

Comment (rabi-siddique) [reply test/e2e/test.utils.js:63]: You can choose any name for your key. I use `user1` for both local and testnet testing to maintain consistency.

File: README.md (hunk 1 above)
Comment (rabi-siddique) [reply README.md:39]: Good point. Let me make the update.

File: test/e2e/test.utils.js (hunk 2 above)
Comment (rabi-siddique) [reply test/e2e/test.utils.js:63]: PS: using the `agd keys add` command.

**Review by mujahidkay — APPROVED** [review]

**Pull request merged**